frontend/public/components/graphs/prometheus-graph.tsx

```diff
@@ -11,20 +11,28 @@ export const getPrometheusExpressionBrowserURL = (urls, queries): string => {
   }
   const params = new URLSearchParams();
   _.each(queries, (query, i) => {
+    const queryWithoutRange = query.endsWith(']') ? query.substring(0, query.lastIndexOf('[')) : query;
```

Comment:
I'm nervous about trying to parse the query string in any way. Is it possible instead of take the query to use in the link as a prop? It would be better to make it explicit imo

Reply:
We can already set a different query than the one which is actually used (we can pass query without range selector). But Im worried we will forget to do so when doing changes to queries/adding new ones..
I'd like to see what @kyoto has to say about parsing it like this. Is there a case where this wouldn't work ? Im not expert in promQL so I'm not sure either.

Reply:
It looks like brackets are only used for range vectors... But I don't think they always have to appear at the end of the query.

https://prometheus.io/docs/prometheus/latest/querying/basics/#range-vector-selectors

I'm just not sure this is a road we want to go down. Could we run into other special cases that are more difficult to handle?

Reply:
These queries are PromQL subqueries, which return range vectors. See https://prometheus.io/docs/prometheus/latest/querying/basics/#subquery

I think the more correct behavior here would be to change the `tab` parameter to `1` for subqueries. Then, the Prometheus UI will open with the "Console" tab (results table tab) visible. The `invalid expression type "range vector" for range query, must be Scalar or instant Vector` message is only for the "Graph" tab, not the "Console" tab.

That would solve our immediate problem with the dashboard links, but to be complete it should also handle the optional `offset` that a subquery may have.